src/components/CompareResults/SubtestsResults/SubtestsResultsMain.tsx

```diff
@@ -55,16 +55,14 @@ function SubtestsResultsMain() {
 
   return (
     <Container className={styles.container} data-testid='subtests-main'>
-      <Await resolve={results}>
-        <header>
-          <SubtestsRevisionHeader header={subtestsHeader} />
-          <div className={styles.content}>
-            <SearchInput onChange={setSearchTerm} />
-            <DownloadButton />
-          </div>
-        </header>
-        <SubtestsResultsTable filteringSearchTerm={searchTerm} />
-      </Await>
+      <header>
+        <SubtestsRevisionHeader header={subtestsHeader} />
+        <div className={styles.content}>
+          <SearchInput onChange={setSearchTerm} />
+          <DownloadButton results={[results]} />
```

Comment:
DownloadButton expects a CompareResultsItem[][] but in the subtests page, `results` is a CompareResultsItem[].